### PR TITLE
adding read messages c4 diagrams

### PR DESCRIPTION
**Pull request by lucamancosu**
#### List of Changes

- [adding read messages c4 diagrams](https://github.com/pagopa/io-messages/commit/b6cc1bb943903fd28a2f256948c2e2c2e9bb5d20)

#### Motivation and Context

Added two c4 diagrams to describe the read message system. The message-list-reader diagram shows how a cityzen can retrieve the message list and which are the system involved components. The mobile-app diagram shows the interactions between the cityzen and the mobile application and the relatad mobile components.

#### How Has This Been Tested?

No tests are needed, just documentaions

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/read-messages/message-list-reader.plantuml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@startuml message-list-reader
+
+!include <C4/C4_Context>
+!include <C4/C4_Container>
+
+LAYOUT_WITH_LEGEND()
+
+title Container Message List Reader System
+
+Person(customer, Cittadino, "Someone in the team working on an issue or component")
```
Comment (Garma00) [inline docs/read-messages/message-list-reader.plantuml]:
```suggestion
Person(customer, Citizen, "Someone in the team working on an issue or component")
```

File: docs/read-messages/mobile-app.plantuml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+!include <C4/C4_Context>
+!include <C4/C4_Container>
+!include <C4/C4_Component>
+
+LAYOUT_WITH_LEGEND()
+
+title Component Mobile App
+
+Person(customer, Cittadino, "Someone in the team working on an issue or component")
```
Comment (Garma00) [inline docs/read-messages/mobile-app.plantuml]:
```suggestion
Person(customer, Citizen, "Someone in the team working on an issue or component")
```

**Review by Garma00 — CHANGES_REQUESTED** [review]

**Review by Garma00 — APPROVED** [review]
GJ

**Comment by Garma00** [comment]
I think we can merge this cc @lucacavallaro @acuiuli

**Review by acuiuli — APPROVED** [review]

**Pull request merged**